### DISCUSSION TITLE
Calibre 7.21.0 + DeDRM 10.0.9 plugin + KFX Input 2.17.0 on MacOS: "Cannot convert .... This book is locked by DRM"

### DISCUSSION THREAD

**Discussion by gpagnon** [noDRM/DeDRM_tools#669] [General]
Hi, I am trying to see whether this works.  I am using the Kindle app on MacOs and I downloaded a book sample from Amazon.jp.  In Calibre 7.21.0 I do:
1) Add books
2) Navigate to the directory where the kindle app stores the files relative to the ebook
3) Select the file *.kfx

Now, calibre shows me that has imported the file with "Formats=KFX-ZIP" and "Folder=Book Files", with no apparent error.

If I now try to convert the book, the processing stops with the message: "Cannot convert... This book is locked by DRM".

Any idea on how to solve this?

thanks in advance for any suggestion!

**Comment by ElleKayEm** [comment]
Most books from the Kindle for Mac app cannot be decrypted at this time.  See this post for other known options:  https://www.mobileread.com/forums/showthread.php?t=361503

**Comment by gpagnon** [comment]
Thank you very much for the info!

On Fri, Nov 8, 2024 at 4:27 PM ElleKayEm ***@***.***> wrote:

> Most books from the Kindle for Mac app cannot be decrypted at this time.
> See this post for other known options:
> https://www.mobileread.com/forums/showthread.php?t=361503
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/noDRM/DeDRM_tools/discussions/669#discussioncomment-11190399>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AFUJAIT45LOGZZS5CY2HRP3Z7TJ7HAVCNFSM6AAAAABRNWGQNSVHI2DSMVQWIX3LMV43URDJONRXK43TNFXW4Q3PNVWWK3TUHMYTCMJZGAZTSOI>
> .
> You are receiving this because you authored the thread.Message ID:
> ***@***.***>
>


-- 
Giuseppe Pagnoni
Dip. Scienze Biomediche, Metaboliche e Neuroscienze
Sezione Fisiologia e Neuroscienze
Univ. di Modena e Reggio Emilia
Via Campi 287
I-41125 Modena, Italy
Tel: +39-059-205-5742
Fax: +39-059-205-5363
